### DISCUSSION TITLE
Spreadsheet Editor is buggy and lacks key features

### DISCUSSION THREAD

**Discussion by skolsocial** [standardnotes/forum#2470] [General] [answered]
**Describe the bug**
The insert row and column features do not work on the spreadsheet editor.

**To Reproduce**
1) Select column or row.
2) Go to Insert Menu
3) Push any insert button - nothing happens.

**Expected behavior**
The Insert should *AT LEAST* happen by pushing buttons.  (Bug report)

I also expect column insert to happen by right-clicking on a selected row or column and seeing a shortcut menu. This does not happen. (Feature request)

**Screenshots**
N/A

**Desktop (please complete the following information):**
* Windows 10
* Google Chrome is up to date
* Version 87.0.4280.66 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
N/A

**Additional context**
Other issues with the spreadsheet editor:
* Missing copy/paste format only (this is table stakes, people)
* A parse data feature would also be nice.

**Comment by jjdrf** [comment]
I second this.
Clicking insert row/column add a row/column but at the end of the spreadsheet. There is no option to add a row/column between existing row/column

**Comment by radianrad** [comment]
Also seeing this behavior. None of the row buttons appear to do anything but add to the end.

**Comment by JaspalSuri** [comment]
Hi everyone,

We weren't able to ship an updated version with a fix for this because inserting a row would break the undo feature. We hope to find a fix for this issue before releasing the updated editor to everyone.

**Comment by myreli** [comment, marked as answer]
We are triaging our older issues, and the Insert menu seems to work as expected on `v 3.149.5` (add/remove columns). Let us know if you run into any other problems.
